index.html

```diff
@@ -155,11 +155,11 @@ <h3>Skills</h3>
 
         
        
-          <form action="">
+          <form action="https://formspree.io/f/xdojqgbe" method="post">
            
-           <input type="text" id="name" name="name" placeholder="Full name">
-            <input type="text" id="email" name="email" placeholder="Email Address">
-            <textarea name="comments" id="comments" cols="30" rows="10">Enter text here</textarea>
+           <input type="text" id="name" name="name" placeholder="Full name" maxlength="30" required>
+            <input type="text" id="email" name="email" placeholder="Email Address" required>
+            <textarea placeholder="Enter text here" name="comments"maxlength="500"></textarea>
```

Comment:
- [ ] **[OPTIONAL]:** I'm sorry the previous reviewer didn't point this out but your `<textarea>` field should be required too. Please consider adding it! 🤓 🛠️

Reply:
Alright thank you